lib/leanplum_api/api.rb

```diff
@@ -210,21 +222,30 @@ def development_connection
 
     # Deletes the user_id and device_id key/value pairs from the hash parameter.
     def extract_user_id_or_device_id_hash!(hash)
-      user_id = hash.delete(:user_id)
-      device_id = hash.delete(:device_id)
+      user_id = hash.delete(:user_id) || hash.delete(:userIid)
+      device_id = hash.delete(:device_id) || hash.delete(:deviceId)
```

Comment:
if you're going to accept `deviceId`, that should be documented in the readme...  not sure why we should accept it though?

Reply:
leanplum api has deviceId, userId. the gem converts device_id for you, but its confusing to have device_id, deviceModel in the same hash.